pygfx/geometries/_text.py

```diff
@@ -638,6 +638,8 @@ def _ensure_text_block_count(self, n):
                 block = self._text_blocks.pop(-1)
                 block._clear(self)
 
+        self.positions.draw_range = 0, n
```

Comment:
is this needed if `n == len(self._text_blocks)`?

Reply:
Yes, because `_text_blocks`  is exactly the size of the number of blocks in used, while `positions` is sized with factors of two.

For the text rendering itself, the `draw_range`  does not have to be set. But it helps when the `positions` buffer is also used to render points/lines, as we do in the ruler. Will add a comment to clarify this.